maryamariyan/runtime
# porting-primitives

-  [x] Flat copy Primitives into runtime repo

   -  [x] Renames test folder to tests

   -  [x] Header license update on cs files

   -  [x] Fix compile for ref project

   -  [x] Fix compile for src project
    - Find references (ILSpy)
    - Fix for all target frameworks (NC and NS2.1)
    - Add shared file (todo: move to common)
    - (Remove extra blank lines)
    [TODO]
    - reduce NoWarn items)

   -  [x] Fix compile for test project

   -  [x] setup packaging
    - Add IsAspNetCoreApp to get separate packaging path
    [TODO]
    - Fix full path
    - Verify package in specified location
    - Verify strong name
    - Use PreReleaseVersionLabel in version.props

   -  [x] Add solution file
    [TODO]: 
    - add pkg folder too
